home/utils.py

```diff
@@ -365,7 +380,9 @@ def get_side_panel_width(ws, column_heading):
             if next_row["viber_body"] not in ["", None]:
                 viber_messages.append(next_row["viber_body"])
 
-        contentpage = add_web(row=row)
+        exiting_contentpage = ContentPage.objects.filter(slug=slug).first()
```

Comment:
I think this will throw an ObjectNotFound error or something if there isn't a page with the given slug 🤔

Reply:
@KaitCrawford I did test this using a slug that was not in the database and it did work and successfully created the page with that slug

Reply:
ok